src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -372,7 +373,7 @@ void GNAPlugin::ImportFrames(
                 auto dst = reinterpret_cast<int8_t*>(ptr_dst);
                 copyInputData(dst, src, num_frames, num_group, num_vector_elements, num_vector_stride, orientation, scaleFactor);
             }
-        } else if (input_precision.size() == 4) {
+        } else if (input_precision.size() == Precision::FP32) {
```

Comment:
Maybe refactor it a little bit (since `if` and `else` contains the same code)

Reply:
done